test/src/tasks/publish-task.test.ts

```diff
@@ -91,9 +117,191 @@ describe(`Run publish task - ${testAppDir}`, function() {
       { path: 'contracts/TestContract.sol' },
       'artifact.json does not include expected data'
     )
+  }
+
+  /**
+   * Test util to fetch the root account
+   * @param bre
+   */
+  async function getRootAccount(
```

Comment:
Maybe move this to test-helpers?

Reply:
Fixed with https://github.com/aragon/buidler-aragon/pull/89/commits/6140156bede94b83e63cc03532839f7291e0712b